protostar-cli/src/lib.rs

```diff
@@ -80,3 +82,15 @@ pub fn get_account<'a>(
 
     Ok(account)
 }
+
+pub fn get_block_id(value: &str) -> Result<BlockId> {
+    // todo: add more block ids (hash, number)
+    match value {
+        "pending" => Ok(BlockId::Tag(Pending)),
+        "latest" => Ok(BlockId::Tag(Latest)),
+        _ => Err(anyhow::anyhow!(
+            "No such block id {}! Possible values are pending and latest for now.",
+            value
```

Comment:
I think you can use `anyhow!` macro here.

Reply:
Am I not using it? :hmmm: